client/my-sites/activity/filterbar/type-selector/activity-type-selector.jsx

```diff
@@ -0,0 +1,87 @@
+import { localize } from 'i18n-calypso';
+import { useQuery } from 'react-query';
+import { connect } from 'react-redux';
+import wpcom from 'calypso/lib/wp';
+import { updateFilter } from 'calypso/state/activity-log/actions';
+import { filterStateToApiQuery } from 'calypso/state/activity-log/utils';
+import { recordTracksEvent, withAnalytics } from 'calypso/state/analytics/actions';
+import fromActivityTypeApi from 'calypso/state/data-layer/wpcom/sites/activity-types/from-api';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+import { TypeSelector } from './type-selector';
+
+const ActivityTypeSelector = ( props ) => {
+	const { translate } = props;
+
+	return <TypeSelector { ...props } title={ translate( 'Activity type' ) } />;
```

Comment:
In production, `'Activity type'` is written with a capitalized `T`. In other words, it reads `'Activity Type'`.

Reply:
Done